docs/source/index.rst

```diff
@@ -15,6 +15,7 @@ The AiiDA common workflows project provides computational workflows, implemented
 The distinguishing feature is that the interfaces of the AiiDA common workflows are unique, independent of the quantum engine that is used underneath to perform the material property simulations.
 These common interfaces make it trivial to switch from quantum engine.
 In addition to the common interface, the workflows provide input generators that automatically define the required inputs for a given
+task.
```

Comment:
Let's keep every sentence on a single line please, and not do any hard-wrapping.